app/views/workflows/edit.html.erb

```diff
@@ -26,7 +26,7 @@
       <p class="help-block">This field is used to indicate to users what level of stability they can expect from the workflow.</p>
     </div>
 
-    <%= render partial: 'assets/license_selector', locals: { resource: @workflow, license_group: :software, default_license: 'Apache-2.0' } %>
+    <%= render partial: 'assets/license_selector', locals: { resource: @workflow, license_group: :software, recommended_licenses: Seek::Config.recommended_software_licenses, default_license: 'Apache-2.0' } %>
```

Comment:
the recommended licenses can be determined inside the partial, based on the license_group. currently this only affects workflows

Reply:
I agree with Stuart